### ISSUE TITLE
[docs] Reference latest OpenTelemetry components version

### ISSUE THREAD

**Issue by rogercoll** [elastic/opentelemetry#40]
Current references in https://github.com/elastic/opentelemetry/blob/main/docs/collector-components.md points to a specific version of the components (0.105.0). OpenTelemetry contrib repository makes weekly releases, should we configure Renovate (or any other automation tool) to keep the versions references up-to-date? Or maybe just reference the main branch?

**Comment by jackshirazi** [comment]
yes we should, it's increasingly out of date and will reference incorrect docs at some point since there will be breaking changes when they go to 1.x of the operator

**Comment by theletterf** [comment]
Is this one still valid? How can we tackle it through automation?

**Comment by rogercoll** [comment]
Closing this as https://www.elastic.co/docs/reference/opentelemetry/edot-collector/components being autogenerated. Thanks!
